src/art/or-yam/meta.json

```
@@ -0,0 +1,5 @@
+{
+    "art_name": "Kind of blue Matrix",
+    "author_name": "or-yam",
+    "author_github_url": "https://github.com/or-yam/"
+}git pull upstream master
```

Should remove unnecessary text

```suggestion
}
```